content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1033,6 +1033,24 @@ and legacy in-tree cloud providers), it sets this annotation on the Node to deno
 set from the command line flag (`--node-ip`). This IP is verified with the cloud provider as valid
 by the cloud-controller-manager.
 
+### volume.alpha.kubernetes.io/node-affinity (alpha) {#volume-alpha-kubernetes-io-node-affinity}
+
+Type: Annotation
+
+Example: `volume.alpha.kubernetes.io/node-affinity: "constrain-with-nodes"`
```

Comment:
```suggestion
Example: `volume.alpha.kubernetes.io/node-affinity: "{}"`
```
As the field no longer works, I wouldn't use a complex example.


If we want one, it could look like:
 `volume.alpha.kubernetes.io/node-affinity: '{"requiredDuringSchedulingIgnoredDuringExecution":{"nodeSelectorTerms":[{"matchExpressions":[{"key":"kubernetes.io/hostname","operator":"In","values":["example-node"]}]}]}}'"`